bentoml/_internal/io_descriptors/pandas.py

```diff
@@ -28,15 +28,18 @@
 if TYPE_CHECKING:
     import pandas as pd
 
+    from bentoml.grpc.v1alpha1 import service_pb2 as pb
+
     from .. import external_typing as ext
     from ..context import InferenceApiContext as Context
 
 else:
+    pb = LazyLoader("pb", globals(), "bentoml.grpc.v1alpha1.service_pb2")
```

Comment:
Should this be using the import util function?

Reply:
oh i forgot to update pandas 😄